src/Areas/ContainerApps/Models/ContainerAppIngress.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace AzureMcp.Areas.ContainerApps.Models;
+
+public class ContainerAppIngress
+{
+    [JsonPropertyName("external")]
+    public bool? External { get; set; }
+
+    [JsonPropertyName("targetPort")]
+    public int? TargetPort { get; set; }
+
+    [JsonPropertyName("fqdn")]
+    public string? Fqdn { get; set; }
```

Comment:
AFAIK this isn't a common abbreviation, can we rename this to FullyQualifedDomainName?

Reply:
Ref: https://learn.microsoft.com/en-us/dotnet/api/azure.provisioning.appcontainers.containerappingressconfiguration.fqdn?view=azure-dotnet-preview#azure-provisioning-appcontainers-containerappingressconfiguration-fqdn